services/QuillLMS/lib/tasks/attach_new_topics_to_activities.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :topics do
+  desc 'Attach topics from new_topics_to_activities csv'
+  task :attach => :environment do
+    topics_table = CSV.parse(File.read("lib/data/new_topics_to_activities.csv"), headers: true)
+    topics_table.each do |row|
+      activity = Activity.find_by(id: row["Activity ID"])
+      next if activity.blank?
+
+      topic_names = [row["Topic 1"], row["Topic 2"], row["Topic 3"]]
+      activity.topic_ids = topic_names.map { |topic_name| Topic.find_by(name: topic_name)&.id }
```

Comment:
Adding `compact` at the end will get rid of potential nils:
`topic_names.map { |topic_name| Topic.find_by(name: topic_name)&.id }.compact`